Sources/SwifterSwift/AppKit/NSImageExtensions.swift

```diff
@@ -58,10 +52,10 @@ public extension NSImage {
         guard let data = tiffRepresentation else { return }
         guard let imageRep = NSBitmapImageRep(data: data) else { return }
 
-        guard let imageData = imageRep.representation(using: type, properties: [.compressionFactor: compressionFactor]) else { return }
+        guard let imageData = imageRep.representation(using: type, properties: [.compressionFactor: compressionFactor])
```

Comment:
humm... this formating style felt really weird to me, I would expect something like breaking at the begining

Reply:
See: https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseonsameline

I personally like `--guardelse same-line`, though I also don't mind sticking to defaults.

![image](https://user-images.githubusercontent.com/526195/89854758-952be980-db49-11ea-84a5-b3c9a68e6e00.png)

Reply:
Let's try `--guardelse same-line`